index.html

```diff
@@ -80,7 +81,7 @@ <h1 class="font">
         </div>
       </section>
       <div class="works" id="portfolio">
-        <div class="showcase-1 showcase">
+        <!-- <div class="showcase-1 showcase">
           <div class="showcase-partition img-content">
             <img
               src="img/Snapshoot-Portfolio.png"
```

Comment:
- [ ] Please do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. Kindly remove the inline comments from your code(**line 84 - 191**). :+1: